docs/SONATA_DEVELOPER_GUIDE.md

```diff
@@ -497,30 +500,38 @@ Table 1: The Layout  of the HDF5 file format for describing nodes.
 
 **node_type_id** -  This is a unique integer for every node type used to associate a node to a node type.  A node type has associated attributes, and a node inherits attributes from its node type.  Attributes associated with a node override attributes inherited from the node type CSV. node_type_id is a unique integer associated with each node type and is used to index all the node type properties associated with a given node with known node_type_id.  These need not be ordered or contiguous, but must be unique.
 
-**model_type** - Has four valid values: , "biophysical"”,“virtual”, “single_compartment”, “point_process”.  In the future, more model_types may be defined. The meaning of each of these model types is as follows.
+**model_type** - Has five valid values: "biophysical"”, "virtual", "single_compartment", "point_process", "point_neuron".  In the future, more model_types may be defined. The meaning of each of these model types is as follows.
```

Comment:
Why do we need to introduce point_neuron?  Can't we just leave it as a "point_process", adding to it then the "nest", "pynn" as valid schema names?

Reply:
I added "point_neuron" since "point_process" has a specific meaning in the context of NMODL. However, I'd also be happy to re-use "point_process" as @markovg suggests. If no-one objects within the next day or so, I'll update the PR accordingly.

Reply:
If point_process can be misunderstood because it looks NEURON specific, I suggest renaming "point_process" to "point_neuron" or even "point"

Reply:
I second changing the name to "point_neuron", which could include NEURON, NEST or PyNN models.

Reply:
Shall we accept the renaming of point_process to point_neuron?

Reply:
OK by me to rename point_process to point_neuron.